framework.go

```diff
@@ -175,6 +175,32 @@ func (f *Framework) ApplyCluster(ctx context.Context, cluster *application.Clust
 		return nil, err
 	}
 
+	ns := cluster.GetNamespace()
+	cm_name := fmt.Sprintf("%s-app-operator-user-values", cluster.Name)
```

Comment:
```suggestion
	configMapName := fmt.Sprintf("%s-app-operator-user-values", cluster.Name)
```

But also as these vars aren't re-used you might as well just directly call them in the ConfigMap below instead of creating the vars